bundles/org.openhab.binding.velbus/README.md

```diff
@@ -136,8 +157,26 @@ The following thing types support setting the dimspeed:
 vmb1dm, vmb1led, vmb4dc, vmbdme, vmbdmi, vmbdmir
 ```
 
+The `vmbdali` thing has 16 virtual light. A virtual light is created from 4 channels of the DALI module.
+This is because an RGBW DALI light is configured on the VMBDALI module with 4 channels (Red channel, Green channel, Blue channel, White channel).  
+
+The configuration is set like this :
+
+```java
+Thing velbus:vmbdali:<bridgeId>:<thingId> [VL1="R,G,B,W"]
+```
+The white channel is optionnal.
+
+e.g.:
+```java
+Thing velbus:vmbdali:<bridgeId>:<thingId> [VL1="CH1,CH2,CH3,CH4", VL2="CH6,CH7,CH5"]
```

Comment:
Since this is an example, fill in the rest of the placeholders as well?

```suggestion
Thing velbus:vmbdali:1:01 [VL1="CH1,CH2,CH3,CH4", VL2="CH6,CH7,CH5"]
```

Reply:
Updated in last commit.